From 3fe574630cdff215ea1f63363081abbc38b3a639 Mon Sep 17 00:00:00 2001
From: Zheng Yang <zhengyang@rock-chips.com>
Date: Mon, 7 May 2018 16:23:46 +0800
Subject: [PATCH] drm/rockchip: inno-hdmi: fix compile warning
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

drivers/gpu/drm/rockchip/inno_hdmi.c:767:5: warning: no previous
prototype for ‘inno_hdmi_audio_config_set’ [-Wmissing-prototypes]

Change-Id: I08596452ee4c4f63afc644d6808c2bb955f643ea
Signed-off-by: Zheng Yang <zhengyang@rock-chips.com>
---
 drivers/gpu/drm/rockchip/inno_hdmi.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/rockchip/inno_hdmi.c b/drivers/gpu/drm/rockchip/inno_hdmi.c
index 56795edd351a..2e037a8efa47 100644
--- a/drivers/gpu/drm/rockchip/inno_hdmi.c
+++ b/drivers/gpu/drm/rockchip/inno_hdmi.c
@@ -764,7 +764,8 @@ static struct drm_connector_helper_funcs inno_hdmi_connector_helper_funcs = {
 	.best_encoder = inno_hdmi_connector_best_encoder,
 };
 
-int inno_hdmi_audio_config_set(struct inno_hdmi *hdmi, struct audio_info *audio)
+static int
+inno_hdmi_audio_config_set(struct inno_hdmi *hdmi, struct audio_info *audio)
 {
 	int rate, N, channel;
 
-- 
2.35.3

